python/pyspark/ml/torch/ssh_manager.py

```diff
@@ -0,0 +1,55 @@
+import os
+import subprocess
+import shutil
+from typing import (List)
+
+from pyspark.ml.torch.utils import write_to_location
+
+class SSHEnvManager:
+    """Is responsible for writing to the known_hosts file, setting up authorized users, and more"""
+    KNOWN_HOSTS = "/root/.ssh/known_hosts"
+    KNOWN_HOSTS_TEMP = "/root/.ssh/known_hosts_temp"
+
+    def __init__(self):
+        self.known_hosts_exists = os.path.exists(SSHEnvManager.KNOWN_HOSTS)
+        if self.known_hosts_exists:
+            shutil.copyfile(SSHEnvManager.KNOWN_HOSTS, SSHEnvManager.KNOWN_HOSTS_TEMP)
+
+    def create_ssh_key(self, ssh_key_path: str):
+        if not os.path.exists(ssh_key_path):
```

Comment:
Prefer not to use negatives in if statements (i.e. please switch the if and else conditions)